tensorflow/lite/delegates/webnn/BUILD

```diff
@@ -31,10 +31,28 @@ cc_library(
         "//tensorflow/lite/c:common",
         "//tensorflow/lite/schema:schema_fbs",
         "//tensorflow/lite/tools/optimize/sparsity:format_converter",
+        "@webnn_native_project//:webnn-native",
+        "@webnn_native_project//:webnn-native-1",
         "@FP16",
     ],
 )
 
+cc_binary(
+    name = "webnn_external_delegate.so",
+    srcs = [
+        "webnn_delegate_adaptor.cc",
+        "webnn_cpp.cpp",
```

Comment:
We should manually copy `webnn_cpp.cpp` to this folder.

Reply:
Could this file be compiled within webnn-native?